scripts-available/CDB_Organizations.sql

```diff
@@ -25,6 +25,58 @@ BEGIN
 END
 $$ LANGUAGE PLPGSQL VOLATILE;
 
+-------------------------------------------------------------------------------
+-- Administrator
+-------------------------------------------------------------------------------
+CREATE OR REPLACE
+FUNCTION cartodb._CDB_Organization_Admin_Role_Name()
+    RETURNS TEXT
+AS 'SELECT current_database() || ''_admin''::text;'
```

Comment:
I prefer the $$ separator, for a moment I doubted if this was the result of a select or a string containing a select...